### PR TITLE
Semantic snippets: allow `cw` snippet in void-returning lambdas

### PR DESCRIPTION
**Pull request by DoctorKrolic**
Fixes: https://github.com/dotnet/roslyn/issues/72266

### CODE REVIEW

File: src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs [hunk 1]
```diff
@@ -20,52 +21,49 @@ namespace Microsoft.CodeAnalysis.Snippets;
 internal abstract class AbstractConsoleSnippetProvider<
     TExpressionStatementSyntax,
     TExpressionSyntax,
-    TArgumentListSyntax> : AbstractStatementSnippetProvider<TExpressionStatementSyntax>
+    TArgumentListSyntax,
+    TLambdaExpressionSyntax> : AbstractSingleChangeSnippetProvider<TExpressionSyntax>
     where TExpressionStatementSyntax : SyntaxNode
     where TExpressionSyntax : SyntaxNode
     where TArgumentListSyntax : SyntaxNode
+    where TLambdaExpressionSyntax : SyntaxNode
```
Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/Snippets/SnippetProviders/AbstractConsoleSnippetProvider.cs:29]:
```suggestion
    where TLambdaExpressionSyntax : TExpressionSyntax
```

**Comment by DoctorKrolic** [comment]
Can I have a review here?

**Comment by DoctorKrolic** [comment]
@akhera99

**Review by akhera99 — APPROVED** [review]
LGTM, thank you for the contribution!

**Pull request merged**